Wrapping/Generators/Python/itk/support/lazy.py

```diff
@@ -45,6 +33,68 @@ def _lazy_itk_module_reconstructor(module_name, state):
 
     return lazy_module
 
+def _ThreadingRLock(*args, **kwargs):
```

Comment:
Perhaps comment out the function, as its invocation is commented out at the moment?